test/fixtures/azure-fixture/src/main/java/fixture/azure/AzureHttpHandler.java

```diff
@@ -355,6 +355,14 @@ public void handle(final HttpExchange exchange) throws IOException {
                                 throw new IllegalStateException("Got multiple deletes in a single request?");
                             }
                             toDelete = blobName;
+                        } else if (Regex.simpleMatch("DELETE /" + account + "/" + container + "/*", line)) {
+                            // possible alternative DELETE url, depending on which method is used in the batch client
+                            String path = RestUtils.decodeComponent(line.split("(\\s|\\?)")[1]);
+                            String blobName = path.split(account)[1];
+                            if (toDelete != null) {
+                                throw new IllegalStateException("Got multiple deletes in a single request?");
+                            }
+                            toDelete = blobName;
```

Comment:
Hi @samxbr, is this what you get when you use the [`BlobBatch#deleteBlob(String blobUrl)`](https://learn.microsoft.com/en-us/java/api/com.azure.storage.blob.batch.blobbatch?view=azure-java-stable#com-azure-storage-blob-batch-blobbatch-deleteblob(java-lang-string)) form (instead of [`BlobBatch#deleteBlob(String containerName, String blobName)`](https://learn.microsoft.com/en-us/java/api/com.azure.storage.blob.batch.blobbatch?view=azure-java-stable#com-azure-storage-blob-batch-blobbatch-deleteblob(java-lang-string-java-lang-string)))

Reply:
Hi @nicktindall, yes exactly. I am using [BlobBatch#deleteBlob(String containerName, String blobName)](https://learn.microsoft.com/en-us/java/api/com.azure.storage.blob.batch.blobbatch?view=azure-java-stable#com-azure-storage-blob-batch-blobbatch-deleteblob(java-lang-string-java-lang-string)) and found that the url is different. Technically I can switch to using [BlobBatch#deleteBlob(String blobUrl)](https://learn.microsoft.com/en-us/java/api/com.azure.storage.blob.batch.blobbatch?view=azure-java-stable#com-azure-storage-blob-batch-blobbatch-deleteblob(java-lang-string)) for my case, but figured it's probably better to just add the support in fixture.